### DISCUSSION TITLE
Non-CIS Reference Architecture with CIS Subscription

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#484] [Help] [answered]
I am a CIS subscriber, but I want to launch a non-CIS Reference Architecture for a project that does not require compliance. However, when I set `UsingCISCompliance: false` in my `reference-architecture-form.yml`, the pre-flight check rejects it. How can I launch a non-CIS Reference Architecture in this case? 

---

<ins datetime="2022-06-23T18:48:40Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108822">Tracked in ticket #108822</a></p>
</ins>

**Comment by zackproser** [comment, marked as answer]
If you are certain that you'd rather deploy a standard non-CIS compliant Reference Architecture despite having paid for the CIS add-on, you can use our "hidden" form field `ForceStandardRefArch`. 

To do so, add the following field at the very bottom of your `reference-architecture-form.yml` file, on its own line, like so: 

...

`ForceStandardRefArch: true`
